src/object-mirrors.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+var ERROR_SOURCE_PROPERTY_NOT_IMPLEMENTED = 'Requiring undefined property on source object';
```

Comment:
And actually could we allow to override the default error message from outside (by passing the message string in BaseXHR constructor for example)?
I liked the fact that we had an explicit message here: https://github.com/streamroot/hlsjs-p2p-wrapper/pull/73/commits/d4b8e82d65fa198297bdd85cc1fb399ad38608ff#diff-50e3aa130a4f97a42ee2cf111c7b1d9dL27